### PR TITLE
replace ngx-md to ngx-markdown (#304)

### PR DESCRIPTION
**Pull request by aprilmay**

### CODE REVIEW

**Comment by HarelM** [comment]
Do you have a screen shot of how the changes look? I'm not even sure where this is shown...?

**Comment by HarelM** [comment]
Just because you asked - the best way to send a new PR is by creating another branch for every PR to avoid seeing "old" commits.

File: projects/showcase/src/app/doc/doc.module.ts [hunk 1]
```diff
@@ -4,11 +4,11 @@ import { FormsModule } from '@angular/forms';
...
 import { MatSelectModule } from '@angular/material/select';
 import { RouterModule, Routes } from '@angular/router';
-import { NgxMdModule } from 'ngx-md';
-import 'prismjs/components/prism-typescript';
-import 'prismjs/prism';
+import { MarkdownModule } from 'ngx-markdown';
 import { LayoutModule } from '../shared/layout/layout.module';
 import { DocComponent } from './doc.component';
+import { HttpClient, HttpClientModule } from '@angular/common/http';
+import 'prismjs/components/prism-typescript.min.js';
```
Comment (HarelM) [inline projects/showcase/src/app/doc/doc.module.ts:11]: Shouldn't this line be inside the `angular.json` file in the scripts?

**Comment by aprilmay** [comment]
ngx-markdown is only used for the showcase/doc. Not sure if it would provide any visual difference from ngx-md, but the latter supports Angular up to v7.

**Comment by HarelM** [comment]
There's no question that a dead package should be replaced.
Please just answer the question above and I'll merge it and release a version.
Thanks again for all the hard work!

File: projects/showcase/src/app/doc/doc.module.ts (hunk 1 above)
Comment (aprilmay) [reply projects/showcase/src/app/doc/doc.module.ts:11]: I just applied the same change from upstream. Actually i quite don't understand the need for this `prismjs-typescript` import.
Moreover it seems that the showcase/doc app works equally well without it.
But i'd leave it like that, just for the sake of keeping minimum changes with upstream.

Comment (HarelM) [reply projects/showcase/src/app/doc/doc.module.ts:11]: Fair enough, worth opening an issue with the upstream repo to understand what this is for.
Thanks again!!!

**Pull request merged**